src/Lecture1.hs

```diff
@@ -119,4 +126,14 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = if null list then "10 is greater than 0 elements and lower than 0 elements" else go 0 0 list
+    where
+        go :: Int -> Int -> [Int] -> String
+        go count equal tempList
+          | null tempList = show n  ++ " is greater than " ++ show count ++ " elements and lower than " ++ show (length list - count - equal) ++ " elements"
+          | head tempList > n = go count equal (tail tempList)
+          | head tempList == n = go count (equal + 1)  (tail tempList)
```

Comment:
That's a correct implementation 🔝 

I mentioned during the lecture that `length` is a slow function and ideally it's better to avoid it. Here, instead of counting the number of equal elements, you can count the number of elements lower than the given element. So you won't need to subtract this from `length` 🙂 

Otherwise, it's a perfect recursive function 👌🏻